README.rst

```diff
@@ -1133,6 +1133,16 @@ We can also query on access type using its numeric value, which will seem more n
 >>> from irods.models import DataAccess, DataObject, User
 >>> data_objects_writable = list(session.query(DataObject,DataAccess,User)).filter(User.name=='alice',  DataAccess.type >= MODIFY)
 
+One last note on permissions:  The <session>.permissions.get(...) method produces inconsistent results when the
+parameter 'report_raw_acls' is allowed to default to False.  Specifically, collections will exhibit the "non-raw-ACL" behavior of
+reporting individual member users' permissions as a by-product of group ACLs, whereas data objects will not.
+
+In release v1.1.6, we move to correct this inconsistency by introducing the synonym <session>.acls that acts almost identically 
+like <session>.permissions, the exception being that in <session>.acls.get(...) method uses a default value of code:`True` rather
```

Comment:
since "exception" is a no-no word here... please consider the following instead...

"the exception being that in" -> "except that the"

Reply:
Fixed it.